gates/cnot.md

```diff
@@ -7,11 +7,11 @@ groups:
 arity: 2
 ---
 
-The Hadamard gate is one of the fundamental quantum gates, enabling qubits to enter a state of superposition.
+The Hadamard gate is one of the fundamental quantum gates, enabling qubits to enter a state of superposition. {.gate-description}
```

Comment:
One way to get around needing this could be to move more stuff in to the yaml header. Which, IIRC you suggested something like this very quickly after seeing the project (Like having a JSON file for each gate, so there is a strict schema as to what is required).

I still like the idea of having some free form area for content, but I think you're right that most stuff can be defined in the header, so there is more availability to style items without the use of markdown attributes (probably foreign to most people who might contribute).

If you agree, I can move most things to the header (description, notations, matrix, and group), and then the table can be moved into nunjucks as well.